Project/Source Files/Screens/Search/SearchViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SearchTabViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
+
+    private var customView: SearchView {
+        return view as! SearchView
+	}
+    private var searchController: UISearchController
+    private var searchScopeButtonTitles = ["Title", "Genre"]
```

Comment:
I would say this variable is unnecessary. You use it only once really (print doesn't count). In selectedScopeButtonIndexDidChange method we can just compare the index, and not the title.